app/controllers/volunteer_ops_controller.rb

```diff
@@ -18,6 +18,10 @@ def index
   end
 
   def show
+    if @volunteer_op.nil?
+      render template: 'pages/404', status: 404
+      return
+    end 
```

Comment:
changes in show method are implemented to satisfy new test in custom_error_pages.feature (mimics functionality added to OrganisationsController in previous commit)